src/core/component.js

```diff
@@ -356,6 +356,11 @@ Component.prototype = {
       if (this.isObjectBased) {
         parseProperty(attrValue, this.schema);
       }
+      // Assets need parsing otherwise #asset-id will not be converted in url for this.data (ex gltf-model)
+      if (this.schema.parse.name === 'assetParse') {
```

Comment:
Adding an exception for a specific parser is brittle. We would need to find a more robust way.

Reply:
Maybe we can always call parse function like it happens for components that have schema with more than one property (am I wrong here?) a few lines below the highlighted ones
`parseProperties(attrValue, this.schema, true, this.name);`